test/api.js

```diff
@@ -41,22 +41,31 @@ const databases = {
   },
 };
 
+const jestCmd =
+  'jest --config jest.config.api.js --verbose --runInBand --forceExit --detectOpenHandles'.split(
+    ' '
+  );
+
 const runAllTests = async (args) => {
-  return execa('yarn', ['test:e2e', ...args], {
+  return execa('yarn', [...jestCmd, ...args], {
     stdio: 'inherit',
     cwd: path.resolve(__dirname, '..'),
     env: {
+      // if STRAPI_LICENSE is in the env the test will run in ee automatically
+      STRAPI_DISABLE_EE: !process.env.STRAPI_LICENSE,
```

Comment:
Do we care that there's no longer a way to attempt to run EE tests without a license? It seems ok to me, someone could always pass an invalid license if they want. I just wanted to point it out in case you see any issues.

Reply:
I was wondering. I think if the need comes up we can extend but I don't see it for now